.github/actions/setup-macos/action.yml

```diff
@@ -20,7 +20,7 @@ runs:
   using: "composite"
   steps:
     # github actions overwrites brew's python. Force it to reassert itself, by running in a separate step.
-    # Borrowed from https://github.com/mesonbuild/meson/blob/master/.github/workflows/macos.yml#L87-L92
+    # Borrowed from https://github.com/mesonbuild/meson/blob/aab2533ab4f7f4c16991620b400d71782f89be1c/.github/workflows/macos.yml#L87-L92
```

Comment:
Vincent: Could you also link to https://github.com/actions/setup-python/issues/577? Thanks a lot!

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2225